lib/streamlit/elements/widgets/text_widgets.py

```diff
@@ -195,31 +239,34 @@ def _text_input(
         self,
         label: str,
         value: SupportsStr = "",
```

Comment:
I wonder if this should be written as `SupportsStr | None` for clarity even if technically `None` is an instance of `SupportsStr`

Reply:
Good point 👍  this is also already done for `_text_area`